.github/workflows/ci.yml

```
@@ -252,17 +252,17 @@ jobs:
           # to complete.
           sudo apt-get -qq remove --purge -y man-db  > /dev/null 2> /dev/null
           # Remove older clang before we install
-          sudo apt-get -qq remove   'clang-12*' 'clang-13*' 'clang-14*' 'clang-15*' 'clang-16*' 'llvm-12*' 'llvm-13*' 'llvm-14*' 'llvm-15*' 'llvm-16*' 'lld-12*' 'lld-13*' 'lld-14*' 'lld-15*' 'lld-16*' > /dev/null 2> /dev/null
+          sudo apt-get -qq remove   'clang-12*' 'clang-13*' 'clang-14*' 'clang-15*' 'clang-16*' 'clang-17*' 'clang-18*' 'llvm-12*' 'llvm-13*' 'llvm-14*' 'llvm-15*' 'llvm-16*' 'lld-12*' 'lld-13*' 'lld-14*' 'lld-15*' 'lld-16*' 'lld-17*' 'lld-18*' > /dev/null 2> /dev/null
 
           # Install clang-XXX, lld-XXX, and debootstrap.
-          echo "deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-18 main" |
-            sudo dd of=/etc/apt/sources.list.d/llvm-toolchain-jammy-18.list
+          echo "deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-19 main" |
+            sudo dd of=/etc/apt/sources.list.d/llvm-toolchain-jammy-19.list
```

```suggestion
          echo "deb http://apt.llvm.org/jammy/ llvm-toolchain-noble-19 main" |
            sudo dd of=/etc/apt/sources.list.d/llvm-toolchain-noble-19.list
```
should update it to noble?

---

ubicloud arm runners are on jammy

---

Just so you know, Ubicloud arm64 runners support both jammy and noble. You can specify the Ubuntu version directly, like `ubicloud-standard-16-arm-ubuntu-2204` or `ubicloud-standard-16-arm-ubuntu-2404`.